app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list["Animal"] = []
+
+    def __init__(self, name: str, helth: int = 100) -> None:
+        self.name = name
+        self.health = helth
```

Comment:
The typo in the parameter name `helth` is causing the assignment to `self.health` to be incorrect. It should be `self.health = health`.